modules/AWS/rds/main.tf

```diff
@@ -22,47 +22,49 @@ module "security_group" {
 }
 
 module "db" {
-  source  = "terraform-aws-modules/rds/aws"
-  version = "~> 3.0"
+  source                      = "terraform-aws-modules/rds/aws"
+  version                     = "~> 5.1"
 
-  identifier = var.rds_instance_identifier
+  identifier                  = var.rds_instance_identifier
 
-  create_db_option_group    = false
-  create_db_parameter_group = false
+  create_db_option_group      = false
+  create_db_parameter_group   = false
 
   # All available versions: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts
-  engine               = "postgres"
-  engine_version       = local.engine_version
-  family               = local.family             # DB parameter group
-  major_engine_version = var.major_engine_version # DB option group
-  instance_class       = var.instance_class
+  engine                      = "postgres"
+  engine_version              = local.engine_version
+  family                      = local.family             # DB parameter group
+  major_engine_version        = var.major_engine_version # DB option group
+  instance_class              = var.instance_class
 
-  allocated_storage = var.allocated_storage
-  iops              = var.iops
+  allocated_storage           = var.allocated_storage
+  iops                        = var.iops
 
-  name     = var.db_name
-  username = local.db_master_username
-  password = local.db_master_password
-  port     = 5432
+  db_name                     = var.db_name
+  username                    = local.db_master_username
+  password                    = local.db_master_password
+  create_random_password      = false
```

Comment:
It doesn't harm to set `create_random_password` to `false`, however based on the implementation, it never being used and the value of this property is not important because `password` variable always has a value in this module.

Reply:
@nghazali in the latest version of this module `create_random_password` is set to True by default, and password is ignored in this case:

> Description: Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file. The password provided will not be used if the variable create_random_password is set to true.

Reply:
I am going to rework it and get rid of password generation to let rds module do that for us if password isn't set.

Reply:
Thanks, make sense